vnext/Desktop.UnitTests/WinRTWebSocketResourceUnitTest.cpp

```diff
@@ -85,7 +85,7 @@ TEST_CLASS (WinRTWebSocketResourceUnitTest) {
     rc->Connect({}, {});
     rc->Close(CloseCode::Normal, {});
 
-    Assert::AreEqual({"Expected Failure"}, errorMessage);
+    Assert::AreEqual({"[0x80004005] Expected Failure"}, errorMessage);
```

Comment:
It has been a while since I've seen my friend E_FAIL :D

Reply:
I'm sure we have this issue all over the place, but wouldn't this fail if this is build on a non en-us machine?

Reply:
@dannyvv from what I've been told, the test could fail if run in a non en-us profile (or is the localization at the system level?).

Perhaps I can force the UT project to be en-us.